src/modules/configurator/components/ConfigEdit.tsx

```diff
@@ -0,0 +1,36 @@
+import { ConfigNavbar } from "./ConfigNavbar";
+import { ConfigSidebar } from "./ConfigSidebar";
+import { ConfigSidebarSelect } from "./ConfigSidebarSelect";
+import { CarImageSlide } from "./CarImageSlide";
+import { useConfigEdit } from "../hooks";
+
+export function ConfigEdit() {
+  const { configurations, isDropdownOpen, activeDropdownName } =
+    useConfigEdit();
+
+  return (
+    <section className="relative h-full">
+      <ConfigNavbar
+        model={configurations?.model}
+        productionYear={configurations?.productionYear}
+      />
+      {/*50px i 70px u calc-u su height ConfigNavbar-a.*/}
+      <section className="relative flex max-sm:flex-col min-h-[calc(100%-50px)] sm:min-h-[calc(100%-70px)]">
+        <div
+          className={`${
```

Comment:
Check out clsx tailwind for optional styling.

Reply:
Added `clsx`.